my_game.py

```diff
@@ -74,16 +74,46 @@
 }
 
 
+def create_players(number_of_players: int):
+    """
+    Create the players
+    """
+
+    player_sprite_list = arcade.SpriteList()
+
+    # replace all sprites on layer "players" with actual player objects
+    for i in range(number_of_players):
+        # Creates Player object
+        p = Player(
+            position=(0, 0),
+            max_hp=20,  # FIXME: add some kind of config for the player to avoid magic numbers
+            speed=3,
+            window=None,
+            equipped_weapon=Weapon(type=WeaponType.SWORD_SHORT),
+            scale=SCALING,
+            key_up=PLAYER_KEYS[i]["up"],
+            key_down=PLAYER_KEYS[i]["down"],
+            key_left=PLAYER_KEYS[i]["left"],
+            key_right=PLAYER_KEYS[i]["right"],
+            key_attack=PLAYER_KEYS[i]["attack"],
+        )
+        # Create Player spritelist
+        player_sprite_list.append(p)
+
+    return player_sprite_list
+
+
 class GameView(arcade.View):
     """
     The view with the game itself
     """
     
-    def __init__(self, level):
+    def __init__(self, level, player_sprite_list):
         
         super(GameView, self).__init__()
```

Comment:
Add a _doc string: to the _init()_:
´´´
'''
level: The level number to load
player_sprite_list: The Players to add to the level
'''
´´´